src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -73,6 +73,11 @@ public override bool IsTriggerCharacter(SourceText text, int characterPosition,
 
         private async Task<IEnumerable<CompletionItem>> GetSnippetsForDocumentAsync(Document document, int position, Workspace workspace, CancellationToken cancellationToken)
         {
+            if (!document.SupportsSyntaxTree)
```

Comment:
This is a C# specific completion provider.  The document should always support syntax trees in this case.